clang/lib/Analysis/FlowSensitive/HTMLLogger.css

```diff
@@ -122,10 +122,22 @@ code.line:has(.bb-select):before {
   font-size: x-small;
   flex-grow: 1;
 }
-.value summary {
+.value > summary {
   background-color: #ace;
   display: flex;
-  justify-content: space-between;
+  justify-content: start;
```

Comment:
this should be `flex-start` rather than `start`, but it doesn't matter because `flex-start` is the default, you can just remove it

Reply:
Got it -- done.